### PR TITLE
Adds missing bucket replication config exception

### PR DESCRIPTION
**Pull request by ebozduman**
Creates a custom exception when the configuration file is missing and the `GetBucketReplication` or `RemoveBucketReplication` API runs.

### CODE REVIEW

**Comment by BigUstad** [comment]
@ebozduman Please write a test in `FunctionalTest.cs`. For ex., MakeBucketAsync, call GetReplicationAsync on it. It should throw MissingBucketReplicationConfiguration.

File: Minio/Exceptions/MissingBucketReplicationConfiguration.cs [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Minio.Exceptions
+{
+    public class MissingBucketReplicationConfiguration : MinioException
```
Comment (kannappanr) [inline Minio/Exceptions/MissingBucketReplicationConfiguration.cs:19]: Should the Exception name contain exception?

Comment (ebozduman) [reply Minio/Exceptions/MissingBucketReplicationConfiguration.cs:19]: Yes, good point.
Done.

**Comment by ebozduman** [comment]
> @ebozduman Please write a test in `FunctionalTest.cs`. For ex., MakeBucketAsync, call GetReplicationAsync on it. It should throw MissingBucketReplicationConfiguration.

Yes. Working on it. I also need to add the exception to those APIs that it applies in the doc file, API.md.

**Review by BigUstad — APPROVED** [review]
LGTM

**Comment by BigUstad** [comment]
@ebozduman Please update branch

**Review by kannappanr — APPROVED** [review]
LGTM

**Pull request merged**